src/helpers/commando/commands/track.js

```diff
@@ -8,6 +8,7 @@ if (_.includes(['de', 'fr', 'ja', 'ko', 'ru'], config.locale.language.toLowerCas
 const monsterData = require(monsterDataPath)
 const typeData = require(`${__dirname}/../../../util/types`)
 const formData = require(`${__dirname}/../../../util/forms`)
+const genData = require(`${__dirname}/../../../util/gens`)
```

Comment:
did you forget to 
`git add src/util/gens.json`?

Reply:
Yep, should be here now.